megamek/unittests/megamek/common/GameTest.java

```diff
@@ -0,0 +1,59 @@
+package megamek.common;
+
+import junit.framework.TestCase;
```

Comment:
We would prefer `org.junit.Assert` in line with the existing tests. That is, unless there is a reason to avoid that class in JUnit4.

Reply:
Thanks for your comment. It is changed to `org.junit.Assert` in 
20827f7